tla-io/src/main/scala/at/forsyte/apalache/io/quint/Quint.scala

```diff
@@ -226,7 +226,13 @@ class Quint(moduleData: QuintOutput) {
                       | ${lambda}""".stripMargin
               )
             case QuintLambda(_, params, _, scope) =>
-              val tlaArgs = params.map(p => tla.name(p.name, Quint.typeToTlaType(types(p.id).typ)))
+              // uniquely rename parameters of name "_" to prevent shadowing of nested "_"s
```

Comment:
This is great! Nice fix :)

we should be sure to add changelog entries for these kinds of bug fixes I think. Do you agree?

Reply:
fix in https://github.com/informalsystems/apalache/pull/2615